content/falcon_api_to_gcp_p1.md

```diff
@@ -0,0 +1,115 @@
+Title: Xây dựng API đơn giản với Python và Falcon - Phần 1  
+Category: python  
+Tags: python, gcp, falcon, api, deploy  
+Author: kube.its.me  
+
+
+Xây dựng API bằng Python là việc rất đơn giản. Trong 4 bài hướng dẫn, mình sẽ giúp bạn tạo một API đơn giản bằng Python với [Falcon framework](http://falcon.readthedocs.io/en/stable/) để khoe với cả trái đất này.
+
+## Chuẩn bị:
+1. Tài khoản github hoặc gitlab, bitbucket...  
+2. Cài trên máy:
+    - git
+    - python 3 
+    - virtualenv
+
+## Khởi tạo ứng dụng
+Đầu tiên, tạo một `repository` rỗng trên github, gitlab...bất cứ đâu cũng được và clone `repository` về máy bạn. Ví dụ tên `repository` là `example-api`.  
```

Comment:
toàn bộ phần 1 này không liên quan tới git, nên bỏ đi, cho vào phần deploy chẳng hạn